### ISSUE TITLE
[Feature] 2-4x speed and external player.

### ISSUE THREAD

**Issue by fourtimesspeedplease** [yukino-app/yukino#56]
Tldr: do you mind adding a video speed slider/picker in settings or below the video?(preferably with custom values) or an 'open video with external player' button on android(preferably desktop/linux also) so I can use mpv/vlc 

**Is your feature request related to a problem? Please describe.**
I prefer watching anime at 2-4+ speed depending on the speaker/genre, unless I missed something this is currently not possible on android. 

**Describe the solution you'd like**
Please add a video speed selector under the video or in settings, preferably with the ability to enter custom values. 
Assuming JS works in electron, something like this may work: document.querySelector('video').defaultPlaybackRate = (speed); //https://stackoverflow.com/questions/3027707/how-to-change-the-playing-speed-of-videos-in-html5 
Or, please add an 'open in' or 'share' button with the stream url so I can open the video in mpv/vlc.

Thanks for maintaning this amazing app!
